grafanalib/core.py

```diff
@@ -1139,5 +1559,95 @@ def to_json_data(self):
             'type': SINGLESTAT_TYPE,
             'valueFontSize': self.valueFontSize,
             'valueMaps': self.valueMaps,
-            'valueName': self.valueName
+            'valueName': self.valueName,
+            'tableColumn': self.tableColumn,
+            'error': self.error,
+            'timeFrom': self.timeFrom,
+            'timeShift': self.timeShift,
+        }
+
+    @staticmethod
+    def parse_json_data(data):
+        if 'colors' in data:
+            data['colors'] = [RGBA.parse_json_data(color)
+                              for color in data['colors']]
+        if 'datasource' in data:
+            data['dataSource'] = data.pop('datasource')
+        if 'targets' in data:
+            data['targets'] = [Target.parse_json_data(target)
+                               for target in data['targets']]
+        if 'gauge' in data:
+            data['gauge'] = Gauge.parse_json_data(data['gauge'])
+        if 'sparkline' in data:
+            data['sparkline'] = SparkLine.parse_json_data(data['sparkline'])
+        if 'mappingTypes' in data:
+            data['mappingTypes'] = [Mapping.parse_json_data(map_type)
+                                    for map_type in data['mappingTypes']]
+        if 'height' in data:
+            data['height'] = Pixels.parse_json_data(data['height'])
+
+        return SingleStat(**data)
+
+
+@attr.s
+class Table(object):
```

Comment:
There's another PR that adds `Table`, #81. I've been shamefully slow in reviewing it.